db/schema.rb

```diff
@@ -51,10 +51,10 @@
     t.datetime "updated_at", precision: 6, null: false
     t.integer "reward"
     t.integer "status", default: 0
-    t.integer "category", default: 0
     t.bigint "helper_id"
-    t.float "long"
-    t.float "lat"
+    t.integer "category", default: 0
+    t.float "long", null: false
+    t.float "lat", null: false
```

Comment:
This is because I never migrated my development db when adding these I think, only the test db